util/dvsim/Deploy.py

```diff
@@ -157,9 +157,36 @@ def __post_init__(self):
         self.odir_ln = os.path.basename(os.path.normpath(self.odir))
         self.log = self.odir + "/" + self.target + ".log"
 
+        # Make exports more easily mergeable with the current process' env.
+        self._process_exports()
+
         # If using LSF, redirect stdout and err to the log file
         self.cmd = self.construct_cmd()
 
+    def _process_exports(self):
+        '''Convert 'exports' as a list of dicts in the HJson to a dict.
+
+        Exports is a list of key-value pairs that are to be exported to the
+        subprocess' environment so that the tools can lookup those options.
+        DVSim limits how the data is presented in the HJSon (cannot have a dict
+        of dicts). This method converts a list of dicts into a dict variable,
```

Comment:
I'm not sure this part of the comment is quite right. When you say "dict of dicts", do you mean something like this syntax: `{ { 'foo': 'bar' }, { 'baz': 'qux' } }`? That's not valid json/javascript/hjson (indeed, what would it mean?).

The dvsim restriction that I know about is that we don't want something like `exports: { 'foo': '{foo}' }` (because of the recursive expansion problem that we talked about in #3345). I thought that the current "parse as a list of string that happen to contain colons" approach was a cunning workaround :-)

Also (super-nitty), "HJson" is mis-capitalised.

Reply:
Updated comment to match HJson terminology, which more accurately captures the restriction.